### ISSUE TITLE
Video from VideoPress: Not able to edit thumbnails

### ISSUE THREAD

**Issue by htdat** [Automattic/wp-calypso#38892]
- I am able to replicate the issue for both Jetpack and Simple sites. 
- Not only me seeing this issue. At least one user and another HE  p1579248141000900-slack-jpop-support

#### Steps to reproduce

1. Call https://wordpress.com/media/videos/your-site.com
2. Try to edit video from VideoPress
3. Try to "Edit Thumbnails"

#### What I expected

A player via which I can choose a thumbnail. 

#### What happened instead

Only see a light grey box where the video should be displayed.

#### Browser / OS version

- Chrome Version 79.0.3945.117 (Official Build) (64-bit)
- MacOS Mojave

#### Screenshot / Video

**Before clicking "Edit Thumbnails":**

![Screen Shot on 2020-01-17 at 15:11:42](https://user-images.githubusercontent.com/10045087/72595174-cdfd3a80-393b-11ea-95e3-60c71558ffd1.png)

**After clicking "Edit Thumbnails":**

![Screen Shot on 2020-01-17 at 15:11:56](https://user-images.githubusercontent.com/10045087/72595172-cb9ae080-393b-11ea-8646-ca6d75b13502.png)

**Errors in browser console**

![Screen Shot 2020-01-17 at 15 05 43](https://user-images.githubusercontent.com/10045087/72595505-888d3d00-393c-11ea-8790-fe8343cff48a.png)

#### Context / Source

2640057-zen

**Comment by kriskarkoski** [comment]
I didn't think it was possible to set thumbnails with Jetpack per https://github.com/Automattic/jetpack/issues/422?

**Comment by jenhooks** [comment]
I have one of these in 2644292-zen

**Site:** https://skillacious.com/
**[Debug is green.](https://jetpack.com/support/debug/?url=https://skillacious.com/)**

Specifically, the user says:

> The editor used to work fine. I've tried other browsers and other computers but
the video editor problem is still there.

**Comment by codestor4** [comment]
Customer in 2644292-zen added:

> I just recently noticed this problem within the last two days. Can’t pinpoint it but I use the video editor everyday.

**Comment by htdat** [comment]
> I didn't think it was possible to set thumbnails with Jetpack per Automattic/jetpack#422?

Thanks @kriskarkoski! 
What do you think, @jeherve? I remember that I could edit thumbnails.

**Comment by liviopv** [comment]
Another report on #17905074-hc

**Comment by jeherve** [comment]
Yes, it used to be possible here in the Calypso interface. It is not an option in wp-admin yet though, hence the Jetpack issue.

I can confirm that it's broken on my end in Calypso as well. This may have been caused by D37470-code. @pgk Do you think you could take a look?

**Comment by pgk** [comment]
> This may have been caused by D37470-code.

@jeherve my first thought would be it is not related, that change was about persisting a user's hd preference

**Comment by htdat** [comment]
2640190-zen
Originally from 4688-gh-jpop-issues

**Comment by thabotswana** [comment]
Another report on 18058018-hc

**Comment by simison** [comment]
@pgk @jeherve 

> my first thought would be it is not related, that change was about persisting a user's hd preference

Did you test by reverting that change to confirm it's not caused by it?

**Comment by pgk** [comment]
> Did you test by reverting that change to confirm it's not caused by it?

@simison yes. The problem was still there. 

I tested a _fresh_ rebuild of the player (without changes) and that seems to fix it. Should be ok now.

**Comment by gwwar** [comment]
> I tested a fresh rebuild of the player (without changes) and that seems to fix it. Should be ok now.

@pgk is this still an issue, and if so are you working on this?

**Comment by htdat** [comment]
Thanks for everyone involved here. I've tested on my site and it's working as expected again.

**Comment by pgk** [comment]
> @pgk is this still an issue, and if so are you working on this?

@gwwar it is fixed in r201884-wpcom. Closing

**Comment by pranali333** [comment]
2640190-zen

User came back with the issue.

> This is for the latest video: (https://videos.files.wordpress.com/QS41TKA1/chemical-weapon-attack-on-syrian-soldiers-deraa-syria-tel-el-jabya-near-golan-heights_hd.mp4) 
> 
> Although I can see that some thumbnails were generated in the Media library (images) but not showing here:
> 
> And this one when I click Edit Thumbnail:
> 
> This video was working yesterday just fine with the thumbnail showing and ability to change it, now having the same issue: (for this video: https://videos.files.wordpress.com/bEu620nW/intercontinental-wars-the-open-conrfrontation-part-2-of-2_hd.mp4)
> 


I tested the issue and here are my results:
Note: I can access the video directly via link.
https://videos.files.wordpress.com/QS41TKA1/chemical-weapon-attack-on-syrian-soldiers-deraa-syria-tel-el-jabya-near-golan-heights_hd.mp4

**1) Tested with chrome - NOT WORKING FOR ME AND ANOTHER HE**


![](https://snipboard.io/IhFqWQ.jpg+)
Image link: https://snipboard.io/IhFqWQ.jpg

**Omkar's test with Chrome:**

![](https://d.pr/i/YDAptL+)
Image link: https://d.pr/i/YDAptL

**Browser Console:**

![](https://snipboard.io/lJeQzP.jpg+)
Image link: https://snipboard.io/lJeQzP.jpg 

**Note: Chrome is working fine for Ana.**

2) FireFox is fine for all three of us.

![](https://snipboard.io/bE4DMg.jpg+)
Image link: https://snipboard.io/bE4DMg.jpg

**Comment by coder-karen** [comment]
The user wrote back to say that 'On Opera (Version:66.0.3515.21), which is my default browser for this site, the above issue is happening, I just tried the same on Firefox and Chrome and it's working there correctly.'

I downloaded Opera and tested this - but still could not replicate (I could see the thumbnail) when using the edit tools.

Regarding the Chrome testing - I was able to see the thumbnail via Chrome as could another - but two other HEs could not (both in India I believe - IP / country blocking issue for the video specifically?). Slack thread[ for reference](https://a8c.slack.com/archives/C0CMN0V97/p1579772747024800).
